benchmarking/benchmark.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
+import yaml
+import os
+import h5py
+from pyradiosky import write_healpix_hdf5
+
+from pyuvsim.simsetup import _write_layout_csv, freq_array_to_params
+
+
+def settings_setup(settings_file, outdir=None):
+    """ Parse settings file and make other variables."""
```

Comment:
Please flesh out the docstrings in this module with parameters and returns sections.

Reply:
I've added parameters and returns to the docstrings.